### PR TITLE
Support plugin_config section in conf file

### PR DESCRIPTION
**Pull request by dingus9**
Enables a dict to configure plugin specific options.

Plus version bump

### CODE REVIEW

File: myui/__init__.py [hunk 1]
```diff
@@ -6,7 +6,11 @@
 import tornado.ioloop
 import tornado.options
 from types import ModuleType
+import logging
 
+access_log = logging.getLogger("tornado.access")
+app_log = logging.getLogger("tornado.application")
+# gen_log = logging.getLogger("tornado.general")
```
Comment (teriyakichild) [inline myui/__init__.py:13]: should this be commented?

File: myui/__init__.py [hunk 2]
```diff
@@ -15,14 +19,37 @@ def __init__(self,handlers,settings):
 class BaseHandler(tornado.web.RequestHandler):
     pass
 
+def parse_log_file_option(option):
```
Comment (teriyakichild) [inline myui/__init__.py:22]: This doesn't look necessary unless the TODO, "Get application log config working", was working.

File: myui.conf.example [hunk 3]
```diff
@@ -5,3 +5,23 @@ debug=True
...
+  'plugin': {
+    'session': {
+      'max_age_days': .6,
+      'expires_days': None
+    },
+    'auth': {
+      'method': 'ldap',
+      'ldap': {
+        'uri': 'ldaps://auth.example.com:636/',
+        'baseDN': ''ou=users,o=company',
```
Comment (teriyakichild) [inline myui.conf.example]: there is an extra single quote

Comment (dingus9) [inline myui.conf.example]: Oops didn't notice, I'm using a dev config file

File: myui/__init__.py (hunk 1 above)
Comment (dingus9) [inline myui/__init__.py:13]: Decided not to use it, it's more of an internal logger used by tornado, but we may want it in the future so I left it in as more of a note

**Pull request merged**